Makefile

```diff
@@ -1044,6 +1044,13 @@ u-boot-pistachio-nor.img: u-boot-dtb.img u-boot-spl-pistachio.bimg FORCE
 	@dd if=u-boot-spl-pistachio.bimg of=$@ bs=4K count=128 conv=notrunc
 	@dd if=u-boot-dtb.img of=$@ bs=4K count=256 seek=128 conv=notrunc
 
+ifeq ($(UBOOTRELEASE),)
+	-mv  u-boot-pistachio-nor.img u-boot-$(subst $\",,$(CONFIG_SYS_CONFIG_NAME))-$(UBOOTVERSION).img
+else
+	-mv u-boot-pistachio-nor.img "u-boot-$(subst $\",,$(CONFIG_SYS_CONFIG_NAME))-\
```

Comment:
It seems that UBOOTRELEASE is not set for the first CI build of this PR, which leads to the image being created as "u-boot-pistachio_marduk-2015.10.img", but for subsequent CI builds, UBOOTRELEASE gets a value and an image in the name "u-boot-pistachio_marduk-2015.10-g77f85c3.img" is being generated along with the image from first build.

There are 2 problems:
1. The first build itself should create image with the proper name that include the git commit hash
2. The subsequent builds should be made clean